README.md

```diff
@@ -64,9 +64,10 @@ namespace :message_queue do
   desc "Run worker to consume messages from rabbitmq"
   task consumer: :environment do
     GovukMessageQueueConsumer::Consumer.new(
-      queue_name: "some-queue",
       exchange_name: "some-exchange",
-      processor: MyProcessor.new
+      queue_name: "some-queue",
+      processor: MyProcessor.new,
+      bunny_connection: Bunny.new(RabbitMQConfig.new.from_environment),
```

Comment:
This won't work, since `RabbitMQConfig` is inside the `GovukMessageQueueConsumer`.